ENIGMAsystem/SHELL/Platforms/Win32/WINDOWSmain.cpp

```diff
@@ -49,14 +51,16 @@ namespace enigma_user {
 const int os_type = os_windows;
 }  // namespace enigma_user
 
-namespace enigma  //TODO: Find where this belongs
+namespace enigma
 {
+
 HINSTANCE hInstance;
 HWND hWnd;
-LRESULT CALLBACK WndProc(HWND hWnd, UINT message, WPARAM wParam, LPARAM lParam);
 HDC window_hDC;
 HANDLE mainthread;
 
+LRESULT CALLBACK WndProc(HWND hWnd, UINT message, WPARAM wParam, LPARAM lParam);
```

Comment:
This line might be incorrect for msvc. I vaguely remember fixing it in my cmake pr.

Reply:
What about it is wrong? Looking here it seems correct; we just used a different function name for the proc https://docs.microsoft.com/en-us/previous-versions/windows/desktop/legacy/ms633573(v%3Dvs.85)

Reply:
I was wrong it was a similar line tho https://github.com/enigma-dev/enigma-dev/pull/1490/files#diff-5de129638eaf07e20162090473aa3bc3R65 .